src/WebJobs.Script.WebHost/Properties/Resources.resx

```diff
@@ -283,6 +283,14 @@
   <data name="HostSpecializationTrace" xml:space="preserve">
     <value>Starting host specialization</value>
   </data>
+  <data name="LogHostInitializationSettings" xml:space="preserve">
+    <value>Settings: {{
+  "functionsWorkerRuntime": "{functionWorkerRuntime}",
```

Comment:
Does it make a difference if we were to not define the schema here? What if we just log a string (The string can be a JSON produced from the caller code).

Reply:
I believe @brettsam said it was more efficient to just plug in JSON values like so instead of creating a JSON value ourselves

Reply:
Yeah, in my mind, this would be more efficient during specialization... although honestly we're not talking about much and it's only logged a single time. I'll sync with @aishwaryabh and maybe we can simplify this to touch less files...

Reply:
Just changed into a string!